workgroup/aws-athena-workgroup.json

```diff
@@ -29,6 +29,26 @@
                 "Value"
             ]
         },
+        "SelectedEngineVersion": {
+            "description": "The engine version requested by the user. Possible values are determined by the output of ListEngineVersions, including Auto. The default is Auto.",
+            "type": "string"
+        },
+        "EffectiveEngineVersion": {
+            "description": "Read only. The engine version on which the query runs. If the user requests a valid engine version other than Auto, the effective engine version is the same as the engine version that the user requested. If the user requests Auto, the effective engine version is chosen by Athena. When a request to update the engine version is made by a CreateWorkGroup or UpdateWorkGroup operation, the EffectiveEngineVersion field is ignored.",
```

Comment:
Does this need to be marked as ReadOnly in schema? There is a readOnlyProperties property but I'm not sure if that's only for top-level properties. Was curious if you had looked into this at all

Reply:
done